com.unity.render-pipelines.high-definition/Runtime/Debug/VolumeDebug.cs

```diff
@@ -131,7 +131,8 @@ static public List<Type> componentTypes
                 if (s_ComponentTypes == null)
                 {
                     s_ComponentTypes = VolumeManager.instance.baseComponentTypeArray
-                        .Where(t => !t.IsDefined(typeof(VolumeComponentDeprecated), false))
+                        .Where(t => !t.IsDefined(typeof(HideInInspector), false))
```

Comment:
This is a runtime API. I feel it weird to react to editor attribute HideInInspector.

Reply:
We were using mixing them, because we were not deprecating them actually, and we have them on code behind... About HideInInspector, do you think is better to add another attribute?

Reply:
Well I agree it sound stupid to add another to do the "same". But the naming is kinda not intuitive. I am not sure about that.

@JulienIgnace-Unity what do you think of this?

Reply:
HideInInspector is definitely used by our users in shipped games sadly T_T

Reply:
cc @JulienIgnace-Unity , are you good to go with `HideInInspector`?

Reply:
I feel like it would be a bit overkill to have another one. Since here we are talking about debug stuff, I feel like HideInInspector is actually kind of relevant (inspector == debugger)